packages/langchain-graph-retriever/tests/adapters/test_astra.py

```diff
@@ -0,0 +1,53 @@
+from collections.abc import Iterator
+
+import pytest
+from graph_retriever import Adapter
+from graph_retriever.testing.adapter_tests import AdapterComplianceSuite
+from langchain_core.documents import Document
+from langchain_core.embeddings import Embeddings
+
+
+class TestAstraAdapter(AdapterComplianceSuite):
```

Comment:
this is neat! I like how we just test the `AdapterComplianceSuite` for the specific adapter.